Microsoft.Toolkit.Uwp.SampleApp/Common/SampleCommand.cs

```diff
@@ -21,7 +21,7 @@ public class SampleCommand : DelegateCommand
     {
         public string Label { get; set; }
 
-        public SampleCommand(string name, Action action)
+        public SampleCommand(string name, Action<object> action)
```

Comment:
I haven't found where this change is used yet, can you point it out?

Reply:
https://github.com/Microsoft/UWPCommunityToolkit/pull/1896/files#diff-3cb108bbfe147427fb6b9a3f93d89cc7R118